force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -2748,6 +2748,110 @@
         <shortDescription>Title of the Schedule Visualizer Component</shortDescription>
         <value>Upcoming Installments</value>
     </labels>
+    <labels>
```

Comment:
@sam-knox Here is all the custom labels that need to be reviewed. The 3 attached pictures is where most of the labels are used
<img width="1249" alt="Screen Shot 2022-01-18 at 11 17 23 AM" src="https://user-images.githubusercontent.com/57271120/150004122-55af9e3a-d31f-4692-9589-4f46ef08f89c.png">
<img width="1244" alt="Screen Shot 2022-01-18 at 11 17 41 AM" src="https://user-images.githubusercontent.com/57271120/150004156-7902a791-6bac-482f-9194-b46b174cd7de.png">
<img width="636" alt="Screen Shot 2022-01-18 at 11 18 18 AM" src="https://user-images.githubusercontent.com/57271120/150004179-de2172f7-af81-4512-968c-830fe54fda11.png">

Reply:
@andrewyu-salesforce - Reviewing these labels today.